crdppf/models.py

```diff
@@ -235,39 +233,49 @@ class CHRailwaysConstructionLimits(GeoInterface,Base):
+    class CHAirportSecurityZones(GeoInterface,Base):
+        __tablename__ = 'r108_bazl_sicherheitszonenplan'
+        __table_args__ = {'schema': db_config['schema'], 'autoload': True}
+        idobj = Column(Integer, primary_key=True)
+        geom = GeometryColumn(Geometry(2,srid=srid_))
 
-class CHAirportSecurityZones(GeoInterface,Base):
-    __tablename__ = 'r108_bazl_sicherheitszonenplan'
-    __table_args__ = {'schema': db_config['schema'], 'autoload': True}
-    idobj = Column(Integer, primary_key=True)
-    geom = GeometryColumn(Geometry(2,srid=srid_))
-
-class CHAirportSecurityZonesPDF(GeoInterface,Base):
-    __tablename__ = 'r108_bazl_sicherheitszonenplan_pdf'
-    __table_args__ = {'schema': db_config['schema'], 'autoload': True}
-    idobj = Column(Integer, primary_key=True)
-    geom = GeometryColumn(Geometry(2,srid=srid_))
-    # ch.bazl.sicherheitszonenplan.oereb
-
-GeometryDDL(CHAirportSecurityZonesPDF.__table__)
+    class CHAirportSecurityZonesPDF(GeoInterface,Base):
+        __tablename__ = 'r108_bazl_sicherheitszonenplan_pdf'
+        __table_args__ = {'schema': db_config['schema'], 'autoload': True}
+        idobj = Column(Integer, primary_key=True)
+        geom = GeometryColumn(Geometry(2,srid=srid_))
+        # ch.bazl.sicherheitszonenplan.oereb
 
-class CHAirportProjectZones(GeoInterface,Base):
-    __tablename__ = 'r103_bazl_projektierungszonen_flughafenanlagen'
-    __table_args__ = {'schema': db_config['schema'], 'autoload': True}
-    idobj = Column(Integer, primary_key=True)
-    geom = GeometryColumn(Geometry(2,srid=srid_))
-    
-class CHAirportProjectZonesPDF(GeoInterface,Base):
-    __tablename__ = 'r103_bazl_projektierungszonen_flughafenanlagen_pdf'
-    __table_args__ = {'schema': db_config['schema'], 'autoload': True}
-    idobj = Column(Integer, primary_key=True)
-    geom = GeometryColumn(Geometry(2,srid=srid_))
-    # ch.bazl.projektierungszonen-flughafenanlagen.oereb
+    GeometryDDL(CHAirportSecurityZonesPDF.__table__)
+else:
+    class CHAirportSecurityZones():
+        pass
+    class CHAirportSecurityZonesPDF():
+        pass
 
-GeometryDDL(CHAirportProjectZonesPDF.__table__)
+if 'airport_project_zones' in db_config['restrictions']: 
+    class CHAirportProjectZones(GeoInterface,Base):
+        __tablename__ = 'r103_bazl_projektierungszonen_flughafenanlagen'
+        __table_args__ = {'schema': db_config['schema'], 'autoload': True}
+        idobj = Column(Integer, primary_key=True)
+        geom = GeometryColumn(Geometry(2,srid=srid_))
+        
+    class CHAirportProjectZonesPDF(GeoInterface,Base):
+        __tablename__ = 'r103_bazl_projektierungszonen_flughafenanlagen_pdf'
+        __table_args__ = {'schema': db_config['schema'], 'autoload': True}
+        idobj = Column(Integer, primary_key=True)
+        geom = GeometryColumn(Geometry(2,srid=srid_))
+        # ch.bazl.projektierungszonen-flughafenanlagen.oereb
 
+    GeometryDDL(CHAirportProjectZonesPDF.__table__)
```

Comment:
`GeometryDDL` is only needed if you plan to insert, update or delete data in the table...